.dockerignore

```diff
@@ -0,0 +1,18 @@
+.dockerignore
+.DS_Store
+.editorconfig
+.env
+.env-dist
+.git
+.gitignore
+*.log
+*.orig
+*.sw?
+CODE_OF_CONDUCT.md
+Dockerfile
+docs
+LICENSE
+Makefile
+node_modules
+README.md
+renovate.json
```

Comment:
So what's left? :laughing: Would it be easier to only copy the files you want in the Dockerfile vs excluding all the files you don't want? It also seems more explicit when adding new files down the road.

Reply:
You're absolutely right. I started with one or two ignores and the list just grew without me considering that I could just select the files I want in the Dockerfile.